web/tests/test_chapters.py

```
@@ -0,0 +1,586 @@
+        event = ChapterEvent.objects.create(
+            chapter=chapter,
+            title="RSVP Test Event",
+            description="An event for testing RSVPs",
+            event_type="meetup",
+            start_time=timezone.now() + datetime.timedelta(days=1),
+            end_time=timezone.now() + datetime.timedelta(days=1, hours=2),
+            max_attendees=2,  # Set limit for testing
+            organizer=self.user,
+        )
+        # Create users
+        user1 = User.objects.create_user(username="user1", email="user1@example.com", password="password123")
```

Ruff (0.8.2)

Possible hardcoded password assigned to argument: "password"

(S106)